### PR TITLE
W-16687924-mule-version-note-duke

### PR DESCRIPTION
**Pull request by dukesphere**

### CODE REVIEW

File: modules/ROOT/pages/_partials/acb-runtime-java.adoc [hunk 1]
```diff
@@ -1,25 +1,16 @@
-//
```
Comment (dukesphere) [inline modules/ROOT/pages/_partials/acb-runtime-java.adoc:1]: removed duplicate (same partial in release notes repo)

File: modules/ROOT/pages/mock-data-using-dw-library.adoc [hunk 2]
```diff
@@ -142,14 +142,15 @@ MuleSoft: Deploy to CloudHub
...
+  "runtime": "4.4.0", //<1>
   "workerSize": 0.1,
-  "applicationName": "mock-dw-library", //<1>
+  "applicationName": "mock-dw-library", //<2>
   "workers": 1,
   "autoStart": true
 }
 --
 [calloutlist]
+.. The `runtime` version is the Mule runtime version you select when creating the integration. For more information, see
```
Comment (chris-campbell9) [inline modules/ROOT/pages/mock-data-using-dw-library.adoc]: Did the last sentence get cut off? "For more information, see ??"

File: modules/ROOT/pages/mock-data-using-dw-library.adoc [hunk 3]
```diff
@@ -33,8 +33,8 @@ image::int-create-integration-scratch.png[]
 .. Under *Create*, select *Empty Project*. 
 .. Select a *Mule runtime* and *Java* version.
 +
-//Info about downloads:
-include::partial$acb-runtime-java.adoc[tags="runtime-java-download"]
+//Info about downloads and versioning:
```
Comment (dukesphere) [inline modules/ROOT/pages/mock-data-using-dw-library.adoc:36]: good catch, @chris-campbell9, fixed

File: modules/ROOT/pages/mock-data-using-dw-library.adoc (hunk 2 above)
Comment (dukesphere) [reply modules/ROOT/pages/mock-data-using-dw-library.adoc]: good catch, @chris-campbell9, fixed

**Review by chris-campbell9 — APPROVED** [review]

**Pull request merged**